submissions/kovalov/html-css-popup/styles.css

```diff
@@ -161,12 +175,12 @@ body {
 	appearance: none;
 	cursor: pointer;
 	outline: none;
-	border: 2px solid transparent;
+	/* border: 2px solid transparent; */
```

Comment:
Avoid adding commented code to the commit.

Reply:
Haven't found commented code line.